### PR TITLE
fix!: Misc breaking changes

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UWPSyncGenerator/Helpers/SymbolMatchingHelpers.cs [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+		{
+			return false;
+		}
+
+		if (uapMethod.Name == "ToString" && uapMethod.Parameters.IsEmpty)
+		{
+			return true;
+		}
+
+		if (uapMethod.Name is "LoadContent" or "Measure" or "Arrange")
```
Comment (Youssef1313) [inline src/Uno.UWPSyncGenerator/Helpers/SymbolMatchingHelpers.cs:163]: Addressing LoadContent by https://github.com/unoplatform/uno/pull/12276.

File: src/Uno.UWPSyncGenerator/Helpers/SymbolMatchingHelpers.cs [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+		return result;
+	}
+
+	private static bool AreEventsMatching(IEventSymbol uapEvent, IEventSymbol unoEvent)
+	{
+		if (uapEvent.ContainingType.Name == "Window")
+		{
+			// TODO: Match API with WinUI.
+			return true;
+		}
```
Comment (Youssef1313) [inline src/Uno.UWPSyncGenerator/Helpers/SymbolMatchingHelpers.cs:113]: @MartinZikmund Would you like to handle this?

File: src/Uno.UWPSyncGenerator/Helpers/SymbolMatchingHelpers.cs [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+		uapField.IsReadOnly == unoField.IsReadOnly &&
+		AreMatching(uapField.Type, unoField.Type);
+
+	private static bool ArePropertiesMatching(IPropertySymbol uapProperty, IPropertySymbol unoProperty)
+	{
+		if (uapProperty.Name == "WindowActivationState" && uapProperty.ContainingType.Name == "WindowActivatedEventArgs")
+		{
+			// TODO: Match API with WinUI.
+			return true;
+		}
```
Comment (Youssef1313) [inline src/Uno.UWPSyncGenerator/Helpers/SymbolMatchingHelpers.cs:133]: @MartinZikmund This as well.

File: src/Uno.UI/UI/Xaml/StateTriggerBase.cs [hunk 4]
```diff
@@ -22,9 +22,9 @@ protected StateTriggerBase()
 			);
 		}
 
-		protected void SetActive(bool active)
+		protected void SetActive(bool IsActive)
```
Comment (jeromelaban) [inline src/Uno.UI/UI/Xaml/StateTriggerBase.cs:25]: This is some very curious casing. Is it this way it's named in winui?

**Review by jeromelaban — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/StateTriggerBase.cs (hunk 4 above)
Comment (Youssef1313) [reply src/Uno.UI/UI/Xaml/StateTriggerBase.cs:25]: Unfortunately, yes:

https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.statetriggerbase.setactive?view=windows-app-sdk-1.3

**Pull request closed without merging**